back-end/functions/auth.py

```diff
@@ -0,0 +1,67 @@
+import json
+from google.oauth2 import id_token
+from google.auth.transport import requests
+import time
+import jwt
+from util.get_secret import get_secret
+
+SECRET_KEY = get_secret("auth", "jwt_secret_key")
+WEB_CLIENT_ID = get_secret("auth", "web_client_id")
+IOS_CLIENT_ID = get_secret("auth", "ios_client_id")
+
+def create_jwt_token(payload):
+    current_time = int(time.time())
+    expiration_time = current_time + 864000 # ten days
+
+    token = jwt.encode(
+        payload,
+        SECRET_KEY,
+        algorithm="HS256",
+        headers={"exp": expiration_time}
+    )
+
+    return token
```

Comment:
I'm not sure if it'd make sense, but do we potentially want to move this to a separate file to import into any serverless function that has a payload we want to generate a jwt for?

Reply:
this should be fixed now :D